mycore-solr/src/main/resources/xslt/solr/indexing/mycoreobject-dynamicfields.xsl

```diff
@@ -5,9 +5,9 @@
   xmlns:fn="http://www.w3.org/2005/xpath-functions"
   xmlns:mcrsolr="http://www.mycore.de/xslt/solr">
   
-  <xsl:import href="xslImport:solr-document-3:solr/indexing/mycoreobject-dynamicfields-3.xsl" />
+  <xsl:import href="xslImport:solr-document:solr/indexing/mycoreobject-dynamicfields.xsl" />
```

Comment:
I thing, the path should also start with `/xsl` or `/xslt` to distinguish version 1 and version 3 XSL templates.
Or does that break to much existing code?

Reply:
Currently, we cann not use resource:xslt/ in xslImport, since the respective URI resolver always adds a resource: on its own (https://github.com/MyCoRe-Org/mycore/blob/main/mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java#L1369)